src/library_syscall.js

```diff
@@ -770,7 +770,7 @@ var SyscallsLibrary = {
   },
   __sys_wait4__proxy: false,
   __sys_wait4: function(pid, wstart, options, rusage) {
-    abort('cannot wait on child processes');
+    return -{{{ cDefine('ENOSYS') }}}; // unsupported feature
```

Comment:
this looks like a merge issue? this landed on master i believe. there are also deps_info changes here that looks unintended.

Reply:
This is not part of the change..  it seems the github gets confused when the upstream of a given PR is merged into master.  I've reuploaded.